### PR TITLE
Fixing styling and Position of Text in Remove Admin Flow

### PR DESCRIPTION
**Pull request by xwsxethan**
This change is [Reviewable](https://reviewable.io/reviews/uproxy/ufo-management-server-flask/108)

### CODE REVIEW

**Comment by xwsxethan** [comment]
Looks like this now:
uforemoveadminnewtext

File: ufo/static/ufoDropdownMenu.html [hunk 1]
```diff
@@ -95,8 +98,7 @@
     <paper-dialog id="removeAdminDialog" with-backdrop>
       <paper-button on-tap="flipToDropdownFlowFromRemoveAdmin" class="header">
         <iron-icon icon="arrow-back" item-icon></iron-icon>
-      </paper-button>
-      <p>{{resources.removeAdminInstructions}}</p>
+      </paper-button> {{resources.removeAdminInstructions}}
```
Comment (blueandgold) [inline ufo/static/ufoDropdownMenu.html]: Move "{{resources.removeAdminInstructions}}" to its own line?

**Comment by blueandgold** [comment]
:+1:

File: ufo/static/ufoDropdownMenu.html (hunk 1 above)
Comment (xwsxethan) [inline ufo/static/ufoDropdownMenu.html]: Sorry about that! I was experimenting with using it inside the button which requires the space after an icon and then the text all on the same line so I didn't change it back here. Done now!

**Pull request merged**